reasoner/controller/RootDisjunctionController.java

```diff
@@ -48,7 +49,9 @@
 
     @Override
     public void initialise() {
+        long start = PerfCounters.getNanos();
         disjunction.conjunctions().forEach(conjunction -> planner().plan(conjunction, new HashSet<>()));
+        processorContext().perfCounters().timePlanningMs.add((PerfCounters.getNanos() - start) / 1_000_000);
```

Comment:
I wonder -do you just want to move the perf counter timing into the `Planner` itself? Then you don't need to be careful about adding it around every planner call.